client/src/views/Game.vue

```diff
@@ -5,7 +5,7 @@
     >
       <!-- Game Players Drawer -->
       <v-navigation-drawer
-          :expand-on-hover="true"
+          :expand-on-hover="switch1"
```

Comment:
If possible, I would name this variable something closer to the label on the toggle?  for example, I mentioned the label to be Lock Players Menu. You could call it `lockPlayersMenu` accordingly.